www/data-plugins.html

```diff
@@ -0,0 +1,340 @@
+<h2>Plugins for Data Modification</h2>
```

Comment:
This page probably shouldn't be called "plugins"… It's just documenting the data modification pipeline, which your app can implement directly or through a plugin. We don't want people to ignore this page just because they aren't writing or using a plugin.

Reply:
I on the other hand reasoned this could be a cachy word so this will make people click on the link. I wanted to make the link look much as _clickbait_ as possible. Simply because people should read that just in case they realize this is something they need even if they don't realize it yet. I felt like calling it like that will make people come to the page. After all, any code that modifies externally the data is sort of a plugin :)

Reply:
Yeah, drawing people's attention to it is a good idea. I'm just worried that "plugin" might do the opposite. The new name looks great!